### ISSUE TITLE
save and compare previous size result

### ISSUE THREAD

**Issue by unional** [ai/size-limit#318]
It would be nice to see the delta. Similar to how webpack does it.

**Comment by ai** [comment]
I agree. But we need somebody who will create the architecture and send PR.

**Comment by dougalg-js-tw** [comment]
isn't this already supported with `--json` and `compareWith` options?
